### PR TITLE
(Problem on Sticks)

### PR DESCRIPTION
**Pull request by ShubhangiShrestha**

### CODE REVIEW

File: Task 1/Problems on Sticks/Solution_1.cpp [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+#include <iostream>
+#include<algorithm>
+using namespace std;
+
+int main()
+{
+    int T,i;        //T=No. of test cases
+    cin>>T;
+    while(T--){
```
Comment (chandrikadeb7) [inline Task 1/Problems on Sticks/Solution_1.cpp:15]: Please follow 4 space indentation.

File: Task 1/Problems on Sticks/Solution_1.cpp [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        cin>>a[i];
+     int count=0;
+     for(i=0;i<n;i++){
+        if(a[i]==0){
+            continue;
+        }
+
+        if(a[i+1]==a[i])
+            count+=0;
+       else
```
Comment (chandrikadeb7) [inline Task 1/Problems on Sticks/Solution_1.cpp:29]: Please follow 4 space indentation

Comment (ShubhangiShrestha) [reply Task 1/Problems on Sticks/Solution_1.cpp:29]: ok

File: Task 1/Problems on Sticks/Solution_1.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            count+=1;
+     }
+    cout <<count<< endl;
+    }
+    return 0;
```
Comment (nimishbongale) [inline Task 1/Problems on Sticks/Solution_1.cpp:34]:
```suggestion
    int T,i;        //T=No. of test cases
    cin>>T;
    while(T--){
    int n;          //n=no. of elements in the array
    cin>>n;
    int a[n];        //a[n]=Array
    for(i=0;i<n;i++)
        cin>>a[i];
     int count=0;
     for(i=0;i<n;i++){
        if(a[i]==0){
            continue;
        }

        if(a[i+1]==a[i])
            count+=0;
        else
            count+=1;
     }
    cout <<count<< endl;
    }
    return 0;
```

**Review by nimishbongale — COMMENTED** [review]
Good work @ShubhangiShrestha! 🚀 

Code formatting is followed very rigorously as @chandrikadeb7 rightly pointed out.

Maybe you can see how some of the other codes are formatted and follow something similar!

**Pull request closed without merging**